packages/@aws-cdk/aws-lambda/lib/function.ts

```diff
@@ -566,14 +581,24 @@ export class Function extends FunctionBase {
     if (props.securityGroup && props.allowAllOutbound !== undefined) {
       throw new Error(`Configure 'allowAllOutbound' directly on the supplied SecurityGroup.`);
     }
-
-    const securityGroup = props.securityGroup || new ec2.SecurityGroup(this, 'SecurityGroup', {
-      vpc: props.vpc,
-      description: 'Automatic security group for Lambda Function ' + this.node.uniqueId,
-      allowAllOutbound: props.allowAllOutbound
-    });
-
-    this._connections = new ec2.Connections({ securityGroups: [securityGroup] });
+	
```

Comment:
Something off about the indentation here, I suspect you're using tabs while the existing file is using spaces. Can you check if you're using the same indentation as rest of the file?

Reply:
yeah, fixing it now